### DISCUSSION TITLE
Google assist translation error

### DISCUSSION THREAD

**Discussion by BobSanScans** [xulihang/ImageTrans-docs#463] [Q&A]
_Apologizes in advance:
I am a beginner to reading through documentation for feature rich softwares, so if it was already pointed out, forgive me
Also, I can't seem to bring up the error/window that I got in the first place but I will do my best to explain:_

Currently, when I use translation assist, the google translation attempts to use https://translate.google.cn/#view=home&op=translate&sl={sourceLang}&tl={targetLang}&text={text}

Unfortunately the link is broken and it just brings it up 
![image](https://github.com/xulihang/ImageTrans-docs/assets/138962945/a91c227e-6293-436d-ae16-4583ed26a628)
(I believe the .cn moved to .com.hk)

I was able to temporarily fix it by just replacing .cn to .com:
![image](https://github.com/xulihang/ImageTrans-docs/assets/138962945/21de35bf-3663-45dd-9a8a-ba36cac3ad0b)
I did this in the window that popped up when running translation assist (unable to reproduce, now it just doesn't do anything for google) but I am not sure how to actually edit the parameters so that it always uses the new .com instead of the .cn (I was looking in preferences, etc but couldn't find where I would be able to edit that)

**Comment by xulihang** [comment]
This is the web machine translation plugin. You can check out this to learn more: https://www.basiccat.org/new-plugin-machine-translation-via-web/

I haven't updated the plugin for several years. If you need a new feature, please open an issue.
